scripts/publish_prod.sh

```diff
@@ -92,12 +89,6 @@ echo
 echo "Bumping the version number and committing the changes"
 yarn version --new-version "$VERSION"
 
-echo
-echo 'Publishing to npm'
-yarn
-yarn build
-yarn publish --new-version "$VERSION"
-
```

Comment:
Sure, will add the message and update confluence.

Reply:
Oh sorry, I didn't realize the gh action runs automatically. I've got a couple of concerns.

First, we used to do a `yarn login` but now we do not. What impact will that have?

Second, for who are unaware that creating the release in gh triggers a gh action to publish the release to npm. This gh action could fail and we wouldn't know about it. It would be ideal to get some kind of alert on the success/failure of this job. Either I think we should make sure that people go watch the gh action until it completes, or we should post a comment to slack with the final result.

Reply:
> First, we used to do a yarn login but now we do not. What impact will that have?

`yarn login` prompts the user to log in using username/email and password. Since we are moving to using NPM token for authentication, `yarn login` is no longer needed.

> Second, ...

Sounds good. I'm going to create a Slack App so the GitHub Action can send messages to `serverless-onboarding-and-enablement-ops`. I'm waiting for Datadog's approval on adding this App.